cmd/bnsd/app/datamigration.go

```diff
@@ -44,4 +51,97 @@ func init() {
 			return nil
 		},
 	})
+
+	datamigration.MustRegister("rewrite username accounts", datamigration.Migration{
+		RequiredSigners: []weave.Address{governingBoard},
+		ChainIDs: []string{
+			"iov-mainnet",
+		},
+		Migrate: rewriteUsernameAccounts,
+	})
 }
+
+func rewriteUsernameAccounts(ctx context.Context, db weave.KVStore) error {
+	governingBoard, err := weave.ParseAddress("cond:gov/rule/0000000000000001")
+	if err != nil {
+		panic(err)
+	}
+
+	now, err := weave.BlockTime(ctx)
+	if err != nil {
+		return errors.Wrap(err, "block time")
+	}
+
+	iov := account.Domain{
+		Metadata:     &weave.Metadata{Schema: 1},
+		Domain:       "iov",
+		Admin:        governingBoard,
+		HasSuperuser: false,
+		AccountRenew: weave.AsUnixDuration(30 * 24 * time.Hour),
```

Comment:
@davepuchyr I have set it to 30 days. What should be the duration?

Reply:
10 years, please